examples/07-EMIT/ComputeChannelSeparation.py

```diff
@@ -0,0 +1,391 @@
+for package in required_packages:
+    if package not in installed_packages:
+        install(package)
+
+# Import required modules
+import plotly.graph_objects as go
+from tqdm.notebook import tqdm
+    
+from matplotlib import pyplot as plt
+plt.ion() # Enables interactive mode so plots show immediately
+plt.show()
+import numpy as np
+
+###############################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode. ``"PYAEDT_NON_GRAPHICAL"``` is needed to generate
+# documentation only.
+# You can set ``non_graphical`` either to ``True`` or ``False``.
+# The ``NewThread`` Boolean variable defines whether to create a new instance
+# of AEDT or try to connect to existing instance of it if one is available.
+
+non_graphical = os.getenv("PYAEDT_NON_GRAPHICAL", "False").lower() in ("true", "1", "t")
+NewThread = False
+desktop_version = "2023.2"
+
+###############################################################################
+# Launch AEDT with EMIT
+# ~~~~~~~~~~~~~~~~~~~~~
+# Launch AEDT with EMIT. The ``Desktop`` class initializes AEDT and starts it
+# on the specified version and in the specified graphical mode.
+
+if desktop_version <= "2023.1":
+    print("Warning: this example requires AEDT 2023.2 or later.")
+    sys.exit()
+
+d = pyaedt.launch_desktop(desktop_version, non_graphical, NewThread)
+emitapp = Emit(pyaedt.generate_unique_project_name())
+
+###############################################################################
+# Create and connect EMIT components
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set up the scenario with radios connected to antennas.
+
+bluetooth, blue_ant = emitapp.modeler.components.create_radio_antenna("Bluetooth Low Energy (LE)", "Bluetooth")
+gps, gps_ant = emitapp.modeler.components.create_radio_antenna("GPS Receiver", "GPS")
+wifi, wifi_ant = emitapp.modeler.components.create_radio_antenna("WiFi - 802.11-2012", "WiFi")
+
+###############################################################################
+# Configure the radios
+# ~~~~~~~~~~~~~~~~~~~~
+# Enable the HR-DSSS bands for the wifi radio and set the power level
+# for all transmit bands to simulate coupling.
+
+wifi_sampling = wifi.get_sampling()
+wifi_sampling.set_channel_sampling(percentage=25)
+for band in wifi.bands():
+    if "HR-DSSS" in band.node_name:
+        band.enabled=True
+        band.set_band_power_level(-50)
+
+# Reduce the bluetooth transmit power
+blue_sampling = bluetooth.get_sampling()
+blue_sampling.set_channel_sampling(percentage=50)
+for band in bluetooth.bands():
+    band.set_band_power_level(-50)
+    
+###############################################################################
+# Load the results set
+# ~~~~~~~~~~~~~~~~~~~~
+# Create a results revision and load it for analysis.
+
+rev = emitapp.results.analyze()
+
+def get_rx_bands(rx_radio):
+    """Return a list of all Rx bands in a given radio.
```

Comment:
This docstrings are not in numpydoc format. Given that there is a "Returns" section, use "Get as the verb for this method and those that follow.